package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "eslint:check": "eslint -c ./.eslintrc.cjs --max-warnings 0 ./",
-    "prettier:check": "prettier -c --no-config ./",
+    "prettier:check": "prettier -c --no-config --ignore-path ./",
```

Comment:
Is this change needed? It was picking up the prettierignore file before file OK before. Was something broken?

Reply:
Nope I meant to remove that. Thanks!